src/main/java/lithium/openstud/driver/core/models/ExamReservation.java

```diff
@@ -189,6 +210,8 @@ public String toString() {
                 ", note='" + note + '\'' +
                 ", ssd='" + ssd + '\'' +
                 ", module='" + module + '\'' +
+                ", attendingModesList='" + attendingModesList + '\'' +
+                ", attendingModeType='" + attendingModeType + '\'' +
```

Comment:
I think it would  be better to update the equals and hashcode method too as the openstud client relies on them quite a bit for handling the app cache